tachyon/base/files/memory_mapped_file.h

```diff
@@ -0,0 +1,170 @@
+// Copyright 2013 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef TACHYON_BASE_FILES_MEMORY_MAPPED_FILE_H_
+#define TACHYON_BASE_FILES_MEMORY_MAPPED_FILE_H_
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include <utility>
+
+#include "absl/types/span.h"
+
+#include "tachyon/export.h"
+#include "tachyon/base/files/file.h"
+// #include "tachyon/base/memory/raw_ptr_exclusion.h"
```

Comment:
Can we remove this line?

Reply:
I'll remain this line to minimize diff and remind me later of uncommenting after we take `raw_ptr_exclusion.h`.